connect/src/protocols/tokenTransfer.ts

```diff
@@ -376,10 +373,7 @@ export class TokenTransfer<N extends Network> implements WormholeTransfer {
     return { chain: transfer.to.chain, address: dstAddress };
   }
 
-  static isAutomatic<N extends Network>(
-    wh: Wormhole<N>,
-    vaa: TokenBridge.VAA<"Transfer" | "TransferWithPayload">,
-  ) {
+  static isAutomatic<N extends Network>(wh: Wormhole<N>, vaa: TokenTransferVAA) {
```

Comment:
And a normal (i.e. no extra/payload3 payload) transfer can never be "automatic" because all relayed transfers carry fee information in their extra payload.

Reply:
should we say "isRelayed"? and is the second one to suggest the type should be narrowed to only those WithPayload?